src/test/resources/test_yaml/android_full.yaml

```diff
@@ -16,3 +17,9 @@ defaults:
     appActivity: LaunchActivity
     appWaitActivity: HomeActivity
     intentCategory: android.intent.category.LEANBACK_LAUNCHER
+    newCommandTimeout: 600
+    idleTimeout: 600
```

Comment:
What is the default value if these are not included in the config?

Reply:
Im still working through the details of what these should be set at . Im currently at values 500 and 270 respectively. Id like the defaults to be nothing and only use them if specified in the yaml file

Reply:
kk, we should test that scenario, remove either `newCommandTimeout` or `idleTimeout` from the configs and verify that it's passing empty or nothing to the `desiredCapabilities`.